index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The AudioWorklet object allows importing user-supplied Javascript code
+          that runs on the audio rendering thread. This processing mechanism
+          ensures the aligned execution of the script code with other built-in
+          <a>AudioNode</a>s in the audio graph. The AudioWorklet object imports
+          and stores the definition of custom audio processor and then an
```

Comment:
"processor" -> "processors"? If you want singular, then "of custom" -> "of a custom audio".